examples-site/react/example-client.tsx

```diff
@@ -13,7 +31,7 @@ async function getComponent(): Promise<ComponentType> {
   const { component, exampleName } = match.groups
 
   return (await import(
-    /* webpackInclude: /\/\w+\/__examples__\/(\w|-)+\.tsx$/ */
+    /* webpackInclude: /\/(\w|-)+\/__examples__\/(\w|-)+\.tsx$/ */
```

Comment:
This was not picking up the back link example `/back-link/__examples__/default.tsx`